src/dashboard/src/components/administration/tests/test_usage.py

```diff
@@ -0,0 +1,53 @@
+import mock
+import os
+
+from django.conf import settings
+from django.test import TestCase
+
+from components import helpers
+
+THIS_DIR = os.path.dirname(os.path.abspath(__file__))
+
+
+class TestUsage(TestCase):
+    fixtures = [os.path.join(THIS_DIR, "../../../../tests/fixtures", "test_user.json")]
+
+    def setUp(self):
+        self.client.login(username="test", password="test")
+        helpers.set_setting("dashboard_uuid", "test-uuid")
+
+    def test_no_calculation(self):
+        for calculate in [None, "False", "false", "no", "0", "random"]:
+            if calculate is None:
+                response = self.client.get("/administration/usage/")
+            else:
+                response = self.client.get(
+                    "/administration/usage/?calculate=%s" % calculate
+                )
+            self.assertFalse(response.context["calculate_usage"])
```

Comment:
This is cool @jraddaoui. The context object is a `django.test.utils.ContextList` I had to look up the [docs](https://docs.djangoproject.com/en/2.2/topics/testing/tools/#django.test.Response.context) but it looks really useful.